Index: backend/src/main/java/mk/ukim/finki/mea_pellicula/config/DatabaseConfig.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package mk.ukim.finki.mea_pellicula.config;\r\n\r\nimport org.springframework.context.annotation.Bean;\r\nimport org.springframework.context.annotation.Configuration;\r\nimport org.springframework.jdbc.core.JdbcTemplate;\r\nimport org.springframework.jdbc.datasource.DriverManagerDataSource;\r\n\r\nimport javax.sql.DataSource;\r\n\r\n@Configuration\r\npublic class DatabaseConfig {\r\n    @Bean\r\n    public DataSource dataSource() {\r\n        DriverManagerDataSource dataSource = new DriverManagerDataSource();\r\n        dataSource.setDriverClassName(\"org.postgresql.Driver\");\r\n        dataSource.setUrl(\"jdbc:postgresql://localhost:5432/mea_peliculla\");\r\n        dataSource.setUsername(\"\");\r\n        dataSource.setPassword(\"\");\r\n        return dataSource;\r\n    }\r\n\r\n    @Bean\r\n    public JdbcTemplate jdbcTemplate(DataSource dataSource) {\r\n        return new JdbcTemplate(dataSource);\r\n    }\r\n}\r\n
===================================================================
diff --git a/backend/src/main/java/mk/ukim/finki/mea_pellicula/config/DatabaseConfig.java b/backend/src/main/java/mk/ukim/finki/mea_pellicula/config/DatabaseConfig.java
--- a/backend/src/main/java/mk/ukim/finki/mea_pellicula/config/DatabaseConfig.java	
+++ b/backend/src/main/java/mk/ukim/finki/mea_pellicula/config/DatabaseConfig.java	
@@ -14,8 +14,8 @@
         DriverManagerDataSource dataSource = new DriverManagerDataSource();
         dataSource.setDriverClassName("org.postgresql.Driver");
         dataSource.setUrl("jdbc:postgresql://localhost:5432/mea_peliculla");
-        dataSource.setUsername("");
-        dataSource.setPassword("");
+        dataSource.setUsername("postgres");
+        dataSource.setPassword("vasko");
         return dataSource;
     }
 
